SETUP.md

```diff
@@ -113,242 +111,6 @@
 
 **NEXT:** if all of the containers show a `Up (healthy)` or `Exit 0` state, then you're ready to start the workshop!
 
-### Instructor-led workshop setup (for trainees)
-
-1. **Prerequisites.**
-
-   The Sensu Go Workshop was designed with modern IT operations, DevOps, and SRE tooling in mind, so we would expect that most users should already have everything they need to complete the workshop.
-   In general, any laptop or other workstation with a `git` client, text editor, and a modern web browser will work great.
-   Please note the following details for more information:
-
-   1. **Workstation**.
-      Workshop users will need a workstation running a recent version of MacOS, Windows, or Linux to participate in this workshop.
-   1. **Supported web browser.**
-      The Sensu Go web app is supported on the latest versions of Chrome, Safari, Firefox, and the Microsoft Edge browser.
-   1. **Compatible text editor.**
-      This workshop requires creation and editing of various configuration files and monitoring code templates.
-      Trainees are encouraged to use a featureful text editor such as [Microsoft Visual Studio Code](https://visualstudio.microsoft.com), [Atom](https://atom.io), or [Submlime Text](https://www.sublimetext.com) (or if you're a `vim` or `emacs` fan, that's fine too).
-      _NOTE: Windows users should avoid using Notepad.exe which can save edited files in incompatible encodings for use with Sensu Go._
-   1. **Optional CLI tools.**
-      The workshop may include examples using certain CLI utilties that trainees may wish to install for convenience (though not required):
-      - `jq` ([website](https://stedolan.github.io/jq/), [downloads](https://stedolan.github.io/jq/download/)).
-
-1. **Download your user `.envrc` or `.envrc.ps1` file.**
-
-   Run the following commands, using the values provided by your instructor for `WORKSHOP_HOSTNAME` and `WORKSHOP_USERNAME`.
-
-   **Mac and Linux users:**
-
-   ```shell
-   WORKSHOP_HOSTNAME=127.0.0.1
-   WORKSHOP_USERNAME=trainee
-   curl -L "http://${WORKSHOP_HOSTNAME}:8000/config/${WORKSHOP_USERNAME}.envrc" -o .envrc
-   ```
-
-   **Windows users (Powershell):**
-
-   ```powershell
-   ${Env:WORKSHOP_HOSTNAME}="127.0.0.1"
-   ${Env:WORKSHOP_USERNAME}="trainee"
-   Invoke-WebRequest -Uri "http://${Env:WORKSHOP_HOSTNAME}:8000/config/${Env:WORKSHOP_USERNAME}.envrc.ps1" -OutFile .envrc.ps1
-   ```
-
-1. **Configure environment variables.**
-
-   **Mac and Linux users:**
-
-   ```shell
-   source .envrc
-   env | grep SENSU
-   ```
-
-   **Windows users (Powershell):**
-
-   ```powershell
-   . .\.envrc.ps1
-   Get-ChildItem env: | Out-String -Stream | Select-String -Pattern SENSU
-   ```
-
-**NEXT:** if you see the expected values output after you run the `env | grep SENSU` command, you're ready for your workshop!
-
-### Instructor-led workshop setup (for instructors)
```

Comment:
@thoward: we should move this documentation somewhere as "optional setup instructions for instructor-led workshops". Will you do that as part of this PR or take care for that as a separate PR?

Reply:
Having said that, it's good to remove it for now since it's already out of date (the shell output examples don't include the new rocketchat bits). :+1:

Reply:
Let's come back to that when we are ready to build out some instructor-focused docs. We will always have this here in the git history as a reference if we need to come back to it.